yarn.lock

```diff
@@ -16168,4 +16168,4 @@ zip-stream@^1.2.0:
     archiver-utils "^1.3.0"
     compress-commons "^1.2.0"
     lodash "^4.8.0"
-    readable-stream "^2.0.0"
+    readable-stream "^2.0.0"
```

Comment:
Hello, so it seems like you copy pasted the `yarn.lock` back into the file, but then removed a newline.

Could you try [checking out the file and reset to the master branch](https://stackoverflow.com/questions/1817766/how-to-revert-to-origins-master-branchs-version-of-file) like: `git checkout origin/master yarn.lock`?

Thank you!

Reply:
@torch2424 Yes, sure!
Hi `yarn.lock` is back from upstream/master as it is.
Previously i've copy paste - yes you was right, sorry for this concern.
Please have a check it again and thanks for you time 😄

Reply:
@torch2424 Just a little reminder 😉
Thank you 😊

Reply:
@torch2424
Guys, please come back here🙂